### PR TITLE
Enable PCI Pass-through and IOMMU as a default kernel parameter

### PR DESCRIPTION
**Pull request by truongmd**
## Description

Enabled PCI Pass-through mode and enables the IOMMU by default

## Why is this needed

Needed to enable ISO support

## How are existing users impacted? What migration steps/scripts do we need?

No expected user impact

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [ ] added unit or e2e tests
- [ ] provided instructions on how to upgrade

### CODE REVIEW

File: installers/osie/main.go [hunk 1]
```diff
@@ -119,6 +119,8 @@ func kernelParams(ctx context.Context, action, state string, j job.Job, s ipxe.S
 	switch j.PlanSlug() {
 	case "c2.large.arm", "c2.large.anbox":
 		s.Args("iommu.passthrough=1")
+        default:
```
Comment (zevweiss) [inline installers/osie/main.go:122]: Indentation looks mismatched (spaces vs. tabs); run `gofmt` to fix I'd expect?

File: installers/osie/main.go [hunk 2]
```diff
@@ -119,6 +119,8 @@ func kernelParams(ctx context.Context, action, state string, j job.Job, s ipxe.S
...
+                s.Args("intel_iommu=on amd_iommu=on iommu=pt")
```
Comment (zevweiss) [inline installers/osie/main.go:123]: I don't think `on` is a recognized value for `amd_iommu` (see https://www.kernel.org/doc/Documentation/admin-guide/kernel-parameters.txt), so we should probably take that one out.

**Review by zevweiss — CHANGES_REQUESTED** [review]

File: installers/osie/main.go [hunk 3]
```diff
@@ -119,6 +119,8 @@ func kernelParams(ctx context.Context, action, state string, j job.Job, s ipxe.S
 	switch j.PlanSlug() {
 	case "c2.large.arm", "c2.large.anbox":
```
Comment (enkelprifti98) [inline installers/osie/main.go:120]: Could we add the recently released "c3.large.arm" here?

Comment (truongmd) [reply installers/osie/main.go]: Added in a following commit

**Pull request closed without merging**